viper.go

```diff
@@ -486,6 +487,8 @@ func (v *Viper) Get(key string) interface{} {
 				val = cast.ToInt(flag.ValueString())
 			case "bool":
 				val = cast.ToBool(flag.ValueString())
+			case "stringSlice":
```

Comment:
As I mentioned in other PR comment. I'm not sure if this is necessary, so I hope you know why this might be correct more than I do. Not only do I forget why I put it there in my diff, but it just plain looks wrong haha :)